recipes/default.rb

```diff
@@ -25,11 +25,17 @@
   action :create
 end
 
+replace_or_add "api_proxy" do
```

Comment:
Shouldn't this go after the execute?

Reply:
The idea was that we could add this before login, and login would use the api_proxy info. But what finally merged into cnspec doesn't actually work that way. So this will move down to after the execute.